src/studylist/StudyListWithData.js

```diff
@@ -28,23 +28,25 @@ class StudyListWithData extends Component {
     .subtract(StudyListWithData.studyListDateFilterNumDays, 'days')
     .toDate();
   static defaultStudyDateTo = new Date();
+  static defaultSearchData = {
+    currentPage: 0,
+    rowsPerPage: StudyListWithData.rowsPerPage,
+    studyDateFrom: StudyListWithData.defaultStudyDateFrom,
+    studyDateTo: StudyListWithData.defaultStudyDateTo,
+    sortData: StudyListWithData.defaultSort,
+  };
 
   componentDidMount() {
     // TODO: Avoid using timepoints here
     //const params = { studyInstanceUids, seriesInstanceUids, timepointId, timepointsFilter={} };
 
-    this.searchForStudies();
+    this.searchForStudies({
+      ...StudyListWithData.defaultSearchData,
+      ...(this.props.filters || {}),
+    });
```

Comment:
I mentioned this in the description but figured inline would make sure it gets noticed. I wonder if we should apply these filters that are passed in on every subsequent search as well. This behavior might feel a little strange.

Thoughts?

Reply:
I think that makes sense. Let's go with that for now and see if users like it? I suppose there should be some way to clear the filters though.

Reply:
Yeah, I guess there are two main ways to work on that:
* Pre-populate the column filters with those in the query parameters
This solves the problem where we want to search the result set, but would allow them to get out of sync with the url.
* Always filter based on what's in the query parameters but provide a new button to clear them
This is probably best long term solution since it keeps the results in sync with the URL but also allows us to clear the query parameters and start fresh.